extract/spine.js

```diff
@@ -21,9 +21,11 @@ const PROPERTIES = {
 export default function spine(xml, tocItem=false) {
   // TODO This will need to change when we use the spine to navgate the book
   // and allow hidden toc items
-  const items = tocItem ?
-    xml.package.spine.itemref.filter(i => i.id !== tocItem) :
-    xml.package.spine.itemref;
+  let items = Array.isArray(xml.package.spine.itemref) ? xml.package.spine.itemref : [xml.package.spine.itemref];
+
+  if (tocItem) {
+    items = items.filter(i => i.id !== tocItem);
```

Comment:
Apparently epubs need to have a toc: "The toc nav is the only navigation aid that has to be included in the EPUB Navigation Document", therefore that `tocItem` won't ever be falsey. We could potentially get rid of the if altogether and just force the spine to be an array and filter it anyway.